.github/workflows/main.yml

```
@@ -19,3 +19,16 @@ jobs:
       - uses: ./
         with:
           token: ${{ secrets.TOKEN }}
+      - name: switching from HTTPS to SSH
+        run: git remote set-url origin git@github.com:ljharb/repo-report-cache.git
+      - name: check for changes
+        run: git status
+      - name: stage changed files
+        run: git add metadata-ossf-score.json
+      - name: commit changed files
+        run: git commit -m "[add] cache metadata-ossf-score.json" -a
+      - name: Push changes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.TOKEN }}
+          branch: rate-limit
```

do we need this action, or can we have:
```suggestion
      - name: Push changes
        run: git push
```
?